ThreeBotPackages/tfgrid_solutions/tfgrid_solutions/chatflows/monitoring_solution_deploy.py

```diff
@@ -0,0 +1,340 @@
+            channel_host=redis_ip_address,
+            channel_port=6379,
+            channel_name="prometheus",
+        )
+
+        self.prometheus_volume = j.sal.zosv2.volume.create(
+            self.reservation,
+            self.nodes_selected["Prometheus"].node_id,
+            size=self.user_form_data["Prometheus Volume Size"],
+            type=self.user_form_data["Prometheus Volume Disk type"],
+        )
+        j.sal.zosv2.volume.attach(container=prometheus_cont, volume=self.prometheus_volume, mount_point="/data")
+
+        # create grafana container
+        grafana_flist = "https://hub.grid.tf/azmy.3bot/grafana-grafana-latest.flist"
+
+        grafana_cont = j.sal.zosv2.container.create(
+            reservation=self.reservation,
+            node_id=self.nodes_selected["Grafana"].node_id,
+            network_name=self.network.name,
+            ip_address=self.ip_addresses["Grafana"],
+            flist=grafana_flist,
+            storage_url=storage_url,
+            disk_type=self.user_form_data["Grafana Root filesystem Type"],
+            disk_size=self.user_form_data["Grafana Root filesystem Size"],
+            env={},
+            interactive=False,
+            entrypoint="",
+            cpu=self.user_form_data["Grafana CPU"],
+            memory=self.user_form_data["Grafana Memory"],
+        )
+        j.sal.zosv2.container.add_logs(
+            grafana_cont, channel_type="redis", channel_host=redis_ip_address, channel_port=6379, channel_name="grafana"
+        )
+
+        metadata = dict()
+        metadata["chatflow"] = self.user_form_data["chatflow"]
+        metadata["Solution name"] = self.user_form_data["Solution name"]
+        metadata["Solution expiration"] = self.user_form_data["Solution expiration"]
+
+        res = j.sal.reservation_chatflow.solution_model_get(
+            self.user_form_data["Solution name"], "tfgrid.solutions.monitoring.1", metadata
+        )
+        reservation = j.sal.reservation_chatflow.reservation_metadata_add(self.reservation, res)
+        self.resv_id = j.sal.reservation_chatflow.reservation_register_and_pay(
+            reservation, self.expiration, customer_tid=j.me.tid, currency=self.network.currency, bot=self
+        )
+
+        j.sal.reservation_chatflow.reservation_save(
+            self.resv_id, self.user_form_data["Solution name"], "tfgrid.solutions.monitoring.1", self.user_form_data
+        )
+
+    @j.baseclasses.chatflow_step(title="Success", disable_previous=True)
+    def success(self):
+        res = f"""\
+            # Your containers have been deployed successfully. Your reservation id is: {self.resv_id}
+            ## Prometheus
+            #### Access container by ```ssh root@{self.ip_addresses["Prometheus"]}``` where you can manually customize the solutions you want to monitor
+            #### Access Prometheus UI through ```{self.ip_addresses["Prometheus"]}/9090/graph``` which is accessed through your browser
```

Comment:
```suggestion
            #### Access Prometheus UI through ```{self.ip_addresses["Prometheus"]}:9090/graph``` which is accessed through your browser
```

Reply:
updated here https://github.com/threefoldtech/jumpscaleX_threebot/pull/764/commits/2c57640372d6d533176a56676396ab87810b0f3b